app/index.html

```diff
@@ -24,6 +24,8 @@
   <script type="text/javascript" src="js/livebus/livebus.factory.js"></script>
 
   <script type="text/javascript" src="js/shared/shared.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.directive.js"></script>
```

Comment:
If it's meant to be a general module like that then it should drop the `uc` prefix from all the names. And perhaps `backgroundImage` for the module name, `background.image.module.js` for the module file name and `background.image.directive.js` for the directive.

And then I would still place the module under the `event` folder and inject it only there because it is not yet used by anywhere else.